src/Place/Canonical/Exception/MuseumPassNotUniqueInCluster.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\Canonical\Exception;
+
+class MuseumPassNotUniqueInCluster extends \Exception
+{
+    public function __construct(int $amount)
```

Comment:
It could be useful to also pass and report the cluster id.

Reply:
Problem here would be that we got rid of the `PlaceCluster` and just give an array of `uuid` to `getCanonical()`, so we don't know the clusterId. I think we could log the clusterId in `MarkPlacesAsDuplicateFromTableCommand` so we can see if this(or another error) happens during the process of a cluster.

Reply:
Good point, that makes me think that `$cluster = $this->duplicatePlaceRepository->getCluster($clusterId);` from the CLI needs to be done inside the `CanonicalService`. The more logic we can put in the service the better.

Reply:
I've made the changes. It think it would also be possible to remove `$clusterIds = $this->canonicalService->getClusterIds();` in `MarkPlacesAsDuplicateFromTableCommand` and just return 1  Associative Arrays with all the canonical places as keys instead of Cluster by Cluster. But since that might take a bit hit on the database, it might be prudent to wait till the datateam has updated their script to no longer use deleted & rejected places?